.github/workflows/publish-core-images.yaml

```
@@ -25,5 +25,8 @@ jobs:
           - component-name: kubectl-delivery
             dockerfile: build/images/kubectl-delivery/Dockerfile
           - component-name: storage-initializer
-             dockerfile: sdk/python/kubeflow/storage_initializer/Dockerfile
-             context: sdk/python/kubeflow/storage_initializer
\ No newline at end of file
+            dockerfile: sdk/python/kubeflow/storage_initializer/Dockerfile
+            context: sdk/python/kubeflow/storage_initializer
```

```suggestion
            dockerfile: sdk/python/kubeflow/storage_initializer/Dockerfile
            context: sdk/python/kubeflow/storage_initializer
            platforms: linux/amd64,linux/arm64,linux/ppc64le
```

Can you apply the same changes to other existing components (kubectl-delivery and training-operator) as well?